app/controllers/dashboard_controller.rb

```diff
@@ -1,7 +1,10 @@
 class DashboardController < ApplicationController
   def index
-    @appendix_i_total_cnt = ShipmentsApiRetriever.api_call('appendix_i').first['meta']['total']
-    @trade_suspensions_total_cnt = ShipmentsApiRetriever.api_call('trade_suspensions').first['meta']['total']
-    @mandatory_quotas_total_cnt = ShipmentsApiRetriever.api_call('mandatory_quotas').first['meta']['total']
+    @issue_by_category = ShipmentsApiRetriever.grouped_call('issue_type')
+    @commodities = ShipmentsApiRetriever.grouped_call('term')
+    @exporting = ShipmentsApiRetriever.grouped_call('exporter', 'exporter_iso')
+    @importing = ShipmentsApiRetriever.grouped_call('importer', 'importer_iso')
+    @species = ShipmentsApiRetriever.grouped_call('taxon', 'appendix')
+    @taxonomy = ShipmentsApiRetriever.grouped_call
```

Comment:
Even though this works well, I think it's a good idea to specify a parameter for this call as well; it helps identifying what this is for and improves readibility.